src/gabc/gabc-write.c

```diff
@@ -231,7 +231,7 @@ static void gabc_print_char(FILE *f, const grewchar to_print)
 {
     if (write_state == GABC_AT_PROTRUSION_FACTOR) {
         write_state = GABC_IN_PROTRUSION_FACTOR;
-        if (to_print == ':') {
+        if (to_print == 'd') {
```

Comment:
This is a bug in the gabc->gabc translation that was made apparent by the coverage testing.  The parser uses `d` for the argument in the internal structure when `<pr>` (no argument) is used.